### PR TITLE
Octez 16.1 packages

### PR DESCRIPTION
**Pull request by killian-delarue**
Packages for Octez v16.1

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
The non-deterministic tests that have been failing since 15.0 are still around:
```
#=== ERROR while compiling tezos-baking-015-PtLimaPt.16.1 =====================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.14.1 | pinned(https://gitlab.com/tezos/tezos/-/archive/v16.1/tezos-16.1.tar.gz)
# path                 ~/.opam/4.14/.opam-switch/build/tezos-baking-015-PtLimaPt.16.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune runtest -p tezos-baking-015-PtLimaPt -j 39
# exit-code            1
# env-file             ~/.opam/log/tezos-baking-015-PtLimaPt-7-e97f51.env
# output-file          ~/.opam/log/tezos-baking-015-PtLimaPt-7-e97f51.out
### output ###
# File "src/proto_015_PtLimaPt/lib_delegate/test/dune", line 32, characters 0-94:
# 32 | (rule
# 33 |  (alias runtest)
# 34 |  (package tezos-baking-015-PtLimaPt)
# 35 |  (action (run %{dep:./main.exe})))
# (cd _build/default/src/proto_015_PtLimaPt/lib_delegate/test && ./main.exe)
# Testing `protocol_alpha'.
# This run has ID `1RPAZOGR'.
# 
#   [OK]          scenario          0   reaches level 5.
#   [OK]          scenario          1   scenario t1.
#   [OK]          scenario          2   scenario t2.
#   [OK]          scenario          3   scenario t3.
#   [OK]          scenario          4   scenario f2.
#   [OK]          scenario          5   scenario m1.
# > [FAIL]        scenario          6   scenario m2.
#   [OK]          scenario          7   scenario m3.
#   [OK]          scenario          8   scenario m4.
#   [OK]          scenario          9   scenario m5.
#   [OK]          scenario         10   scenario m6.
#   [OK]          scenario         11   scenario m7.
#   [OK]          scenario         12   scenario m8.
# 
# ┌──────────────────────────────────────────────────────────────────────────────┐
# │ [FAIL]        scenario          6   scenario m2.                             │
# └──────────────────────────────────────────────────────────────────────────────┘
# Checking provided seed.
# mockup client uses custom bootstrap accounts:
# name:bootstrap1
# sk_uri:unencrypted:edsk3gUfUPyBSfrS9CCgmCiQsTCHGkviBDusMxDJstFtojtc1zcpsh
# amount:4000000;
# name:bootstrap2
# sk_uri:unencrypted:edsk39qAm1fiMjgmPkw1EgQYkMzkJezLNewd7PLNHTkr6w9XA2zdfo
# amount:4000000;
# name:bootstrap3
# sk_uri:unencrypted:edsk4ArLQgBTLWG5FJmnGnT689VKoqhXwmDPBuGx3z4cvwU9MmrPZZ
# amount:4000000;
# name:bootstrap4
# sk_uri:unencrypted:edsk2uqQB9AY4FvioK2YMdfmyMrer5R8mGFyuaLLFfSRo8EoyNdht3
# amount:4000000;
# name:bootstrap5
# sk_uri:unencrypted:edsk4QLrcijEffxV31gGdN2HU7UpyJjA8drFoNcmnB28n89YjPNRFm
# amount:4000000
# 
# Error:
#   the test is taking longer than 60 seconds
#   
# 
# [exception] Alcotest_engine__Core.Make(P)(M).Test_error
#             Raised at Json_encoding.Make.destruct_obj.assoc in file "src/json_encoding.ml", line 499, characters 14-29
#             Called from Json_encoding.Make.destruct_obj.(fun) in file "src/json_encoding.ml", line 527, characters 26-43
#             
# Logs saved to `~/.opam/4.14/.opam-switch/build/tezos-baking-015-PtLimaPt.16.1/_build/default/src/proto_015_PtLimaPt/lib_delegate/test/_build/_tests/protocol_alpha/scenario.006.output'.
#  ──────────────────────────────────────────────────────────────────────────────
# 
# Full test results in `~/.opam/4.14/.opam-switch/build/tezos-baking-015-PtLimaPt.16.1/_build/default/src/proto_015_PtLimaPt/lib_delegate/test/_build/_tests/protocol_alpha'.
# 1 failure! in 194.858s. 13 tests run.
```
Could those be fixed or removed?

Also I know this has only been 2 weeks since i last had a response on this (https://github.com/ocaml/opam-repository/pull/23471#issuecomment-1460195591) but I'd like to make sure this is well under way as the number of people whose opam installation broke because they ran out of inode due to an (unnecessary) influx of packages has been growing (see e.g. https://github.com/ocaml/opam/issues/5484). Is this still on target for the next release?

Closing to free valuable CI time while this is being dealt with.

**Comment by killian-delarue** [comment]
The tests are now removed.

Regarding the packages reduction: Yes it is still a plan to reduce the number of packages. Due to the urgency of `Octez 16.1` it is planned for an other release.

File: packages/tezos-baking-015-PtLimaPt/tezos-baking-015-PtLimaPt.16.1/opam [hunk 1]
```diff
@@ -42,7 +42,7 @@ depends: [
 build: [
   ["rm" "-r" "vendors"]
   ["dune" "build" "-p" name "-j" jobs]
-  ["dune" "runtest" "-p" name "-j" jobs] { with-test & ocaml:native }
+  ["dune" "runtest" "-p" name "-j" jobs] { ocaml:native }
```
Comment (kit-ty-kate) [inline packages/tezos-baking-015-PtLimaPt/tezos-baking-015-PtLimaPt.16.1/opam:45]: I'm sorry but I'm not sure what's "removed" here. If anything you're now running the tests everytime which is worse and not mergeable.

**Comment by kit-ty-kate** [comment]
Closed again to reduce CI load until fixed. Please feel free to reopen when it is.

**Comment by killian-delarue** [comment]
It's now removed

File: packages/tezos-baking-015-PtLimaPt/tezos-baking-015-PtLimaPt.16.1/opam (hunk 1 above)
Comment (killian-delarue) [reply packages/tezos-baking-015-PtLimaPt/tezos-baking-015-PtLimaPt.16.1/opam:45]: Indeed that was niot the way of removing the tests. 
Is the last version ok ? I removed the tests from  ̀tezos-baking-alpha`, `tezos-baking-016` and `tezos-baking-015` which are the packages release in the v16.
If that's not enough I can suggest a seperate PR that do the same thing for Octez v15 ?

**Comment by raphael-proust** [comment]
Removed the label now that the gitlab issue has been resolved. I can't reopen this PR (the button is greyed out) but I think it's ready to be.

**Comment by killian-delarue** [comment]
It would be great if this PR can be merged

**Pull request closed without merging**